### DISCUSSION TITLE
Add a task to github projects

### DISCUSSION THREAD

**Discussion by QuantumLoopHole** [community/community#121343] [Mobile]
Is it possible to add the functionality to add a task to a github project on mobile? I would like to modify some tasks, but it won't let me add one. Please let me know if I am missing anything.

**Comment by niladri-1** [comment]


**Reply by QuantumLoopHole** [reply]
Thank you for the reply, here is what the projects page looks like on my phone:

![Screenshot_20240428_110021_GitHub.jpg](https://github.com/community/community/assets/163697366/c422d236-d1ff-46a1-9bee-4077a9627653)

Please let me know if I am missing something obvious, but I don't see the + icon on here.

**Reply by QuantumLoopHole** [reply]
It is sadly only partially answered. The first response is not correct (as my photo shows).

**Reply by nstringham** [reply]
@niladri-1 can you explain where the "+" button is on the mobile app please?

**Reply by wbudd** [reply]
@niladri-1 It's okay to admit when you mistakenly provide factually incorrect information. It happens. We're all human.

(There is no `+` or 'Add' button on the mobile Github Mobile app's Project views. As demonstrated in the screenshot above, 8 months ago. So no, the problem is very much not solved at all.)

**Reply by Perry-chouteau** [reply]
https://github.com/user-attachments/assets/1971bcd1-4336-47ed-98d9-bf8f6363d71f

You can't create draft task you can only create issue:
```
associate with a repository + (optional) project
```
so the only ways is to:
- create an issue by clicking on the + (main page or on a git repository).
- you can then swipe right throught tags (assignee, label, issue type, millestone, PROJECT)
- click on PROJECT to see a list of your PROJECTS 

> be carefull you can only see the projects of the user the repository is registered in. so if the git repository is in a organisation you won't be able to see your personal project.

Edit: I added a video but it's looks hard to understand cause my finger are not visible :)

**Comment by mark-omarov** [comment]
I am in the same situation - I need a way to quickly manage projects on the go, not only for work but personal things as well. There are many tools like notion, obsidian, todoist, and so on to manage tasks. But, as a developer GitHub Projects stick better with me, plus I can easily create automations with GHA.

So far it seems there's no way to add tasks on mobile app, what a shame. The only option is to head to the website via browser, the experience though is not great and you'll fight the UI. It can be manageable if you need to get things done occasionally, but as a daily driver - nope. 

Tough love I guess. Either hope and wait for GH to add the feature to the app, or use a different tool. Right now I'm looking into Linear, they seem to have a integration with GitHub, developer focused (shortcuts are nice), but their mobile application is in closed beta, so also not fully usable on the go yet. I can't tell if it's good or not, but that's something I'll look into as soon as I get the access.

**Comment by Adnan25z** [comment]
While the GitHub mobile app does not fully support project boards, you can manage issues, which can then be added to the project board. You can create new issues or modify existing ones using the app. To add these issues to a project board, you may need to access the desktop version later.

**Reply by wbudd** [reply]
The problem with that, is that the Github Mobile app only allows creating/updating/removing Issue-specific attributes (e.g., Labels), but no Project specific fields  (e.g., Status, Iteration, or any custom-defined Fields).

Basically, by creating a new issue, you can jot down some text as a quick memo/draft on mobile, but then you have to follow up on a desktop/browser to turn that issue into an item that doesn't make a completely mockery of how the Project was organized in the first place.
